src/main/java/de/triology/cas/services/CesServicesManagerStage.java

```diff
@@ -55,15 +55,18 @@
      * @param name      name of the service
      * @param serviceId regex to match requests against
      */
-    protected void addNewService(String name, String serviceId) {
-        RegexRegisteredService service = new RegexRegisteredService();
+    protected void addNewService(String name, String serviceId, URI logoutUri) {
```

Comment:
You changed all occurrences of addNewService(name,id) to addNewService(name,id,null).
Why you dont add a method with a signature addNewService(String name, String serviceId) which calls the latter without a logoutUri?